### ISSUE TITLE
On Void, `helm-system-packages` asks about missing apt dependencies 

### ISSUE THREAD

**Issue by emacsomancer** [emacs-helm/helm-system-packages#33]
When I run `helm-system-packages` in Emacs on my Void machine, I get the message:

`Dependencies are missing (apt-get, apt-cache, apt-mark), please install them`

I don't see anything in the instructions about specially configuring the system to know that it's Void. 

I also tried `(require helm-system-packages-xbps)` but that only give me one additional command `helm-system-packages-xbps-refresh`, which seems to work, but does not give me an interface to system packages.

**Comment by Ambrevar** [comment]
Thanks for the report.

The xbps package is clearly outdated, but the setup should not be.
Look at https://github.com/emacs-helm/helm-system-packages/blob/master/helm-system-packages.el#L637, you'll see it should try to detect XBPS based on the existence of the `xbps-query` command.

Now it's weird that it matches `dpkg` for you.  Do you have a `dpkg` command on your system?  What does `which dpkg` return in a shell?

**Comment by emacsomancer** [comment]
Yes, I did have the (in the Void repos) `dpkg` package installed (I don't recall what for); removing it resolved the issue (is there a way to manually/forcefully choose which package manager to have `helm-system-packages` attach to? rather than letting it automatically do so? In the case I need the `dpkg` package installed, for instance?)

**Comment by Ambrevar** [comment]
Sadly no.
But there is an easy fix: extract the package manager list from the
`helm-system-packages' function into a variable.
The user could then customize the variable to choose what is looked up.

I can't fix this right now.  Would you be able to send a patch?  I'd
happily merge it then :)

**Comment by emacsomancer** [comment]
Yes, I can work on a patch, though it might take me a little bit to get round to this.
